tracee-ebpf/main.go

```diff
@@ -114,12 +116,30 @@ func main() {
 				fmt.Fprintf(os.Stderr, "warning: could not check enabled kconfig features, trying to continue anyway\n")
 			}
 
-			// try to discover distro by /etc/os-release, fallback to kernel version
+			// OS release information
+
+			osReleaseFilePath, err := checkEnvPath("TRACEE_OSRELEASE_FILE") // useful if users wants to mount host os-release in a container
```

Comment:
We should document this env variable option

Reply:
I have moved this to inside LIBBPF and will document it there after we merge all the osinfo and kernel_features new changes. Just so I have a clear picture on what is the final API merged.